### ISSUE TITLE
Resume from notification bar

### ISSUE THREAD

**Issue by RadioactivePlays** [z-huang/music#1072]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

Resume from notification media card when the app is closed.

### Why do you want this feature?

On Spotify and some other apps, I'm able to resume the last played song from the notification bar while in InnerTune I can't. I disabled battery optimization for it but still doesn't work. I think this feature simply wasn't implement yet and would love to see it implemented. Thanks in advance for your amazing work.

**Comment by thornySoap** [comment]
It's kind of weird but it works for me after I dismiss the not working notification:

https://github.com/z-huang/InnerTune/assets/115401023/5ecd1b81-8e02-4d67-9673-d26764cc37ae

**Comment by RadioactivePlays** [comment]
Yeah just tested and it worked that way. Feature or bug? 🤔 (Don't mind the trash song)

https://github.com/z-huang/InnerTune/assets/19830027/70189a0d-31bc-4a3d-b7d3-e73b1ee5c434
